Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ibmcom/swift-ubuntu:3.0.2
+
+RUN \
+  export DEBIAN_FRONTEND=noninteractive && \
+  apt-get update && \
+  apt-get install curl && \
+  # Allow installation of Node 6: https://nodejs.org/en/download/package-manager/#debian-and-ubuntu-based-linux-distributions \
+  curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash - && \
+  apt-get update && \
+  apt-get install nodejs && \
+  # Yarn's installation instructions did not work, so we are falling back to npm to install it
+  npm install -g yarn;
```

Comment:
[Yarn advise against this](https://yarnpkg.com/en/docs/install#install-via-npm), strongly:

> Note: Installation via npm is generally not recommended. npm is non-deterministic, packages are not signed, and npm does not perform any integrity checks other than a basic SHA1 hash, which is a security risk when installing system-wide apps.

>For these reasons, it is highly recommended that you install Yarn through the installation method best suited to your operating system.

[Better alternative](https://yarnpkg.com/en/docs/install#linux-tab)

```bash
curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -
echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list
apt-get update && sudo apt-get install yarn
```

Do we even need nodejs at all with this?